.moai/specs/SPEC-HOOK-TRACE-FLUSH-001/spec.md

```
@@ -0,0 +1,249 @@
+### 1.2 부수 결함 — 거짓 @MX 주석
+
+`internal/hook/trace/writer.go`의 `Close` 직전 주석은 다음과 같이 주장한다.
+
+```
+// @MX:ANCHOR: [AUTO] Cleanup entry point for all trace consumers; must be called to drain background goroutine
+// @MX:REASON: fan_in=24, all hook handler teardown paths call Close; omitting this call causes the run() goroutine to leak
+```
+
+`fan_in=24`는 거짓이다. 실측 프로덕션 fan_in은 0이다.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)